workflows_core/operator/abstract_operator.py

```diff
@@ -1,23 +1,42 @@
+import json
 import logging
+import numpy as np
+
 from copy import deepcopy
 from abc import ABC, abstractmethod
 
-from typing import List, Optional
+from typing import Any, Dict, List, Optional
 
 from workflows_core.dataset.dataset import Dataset
 from workflows_core.utils.document import Document, DocumentUtils
 
 logger = logging.getLogger(__file__)
 
 
+def is_different(field: str, value1: Any, value2: Any) -> bool:
+    """
+    An all purpose function that checks if two values are different
+    """
+    if "_vector_" in field and isinstance(value1, list) and isinstance(value2, list):
+        element_wise_diff = abs(np.array(value1)) - abs(np.array(value2))
+        sums = np.sum(element_wise_diff)
+        return sums > 0
```

Comment:
Can you please add a few tests here to show this works. Otherwise I call bullshit haha